pyclaragenomics/test/test_cudapoa_bindings.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+from difflib import SequenceMatcher
+import pytest
+import random
+
+from claragenomics.bindings.cudapoa import CudaPoaBatch
+import claragenomics.bindings.cuda as cuda
+
+
+@pytest.mark.gpu
+def test_cudapoa_simple_batch():
+    (free, total) = cuda.cuda_get_mem_info(cuda.cuda_get_device())
+    batch = CudaPoaBatch(10, 0.9 * free)
+    poa_1 = [b"ACTGACTG", b"ACTTACTG", b"ACGGACTG", b"ATCGACTG"]
```

Comment:
Forcing the user to pass in a byte array is rather messy here (as is returning a byte array), can we add support for strings instead?

Reply:
Done